packages/riverpod_analyzer_utils/lib/src/riverpod_ast/resolve_riverpod.dart

```diff
@@ -110,6 +110,12 @@ mixin _ParseRefInvocationMixin on RecursiveAstVisitor<void> {
     final refInvocation = RefInvocation._parse(node, superCall: superCall);
     if (refInvocation != null) {
       visitRefInvocation(refInvocation);
+
+      for (final argument in refInvocation.node.argumentList.arguments
```

Comment:
While this fixes the issue and passes the test, I realize it's probably not the most correct way - happy to update the solution with your guidance.